### DISCUSSION TITLE
unable downloading files

### DISCUSSION THREAD

**Discussion by calidion** [prisma/prisma#13785] [Q&A] [answered]
> Downloading Prisma engines for Node-API for debian-openssl-3.0.x [                    ] 0%Error: Not Found https://binaries.prisma.sh/all_commits/dc520b92b1ebb2d28dc3161f9f82e875bd35d727/debian-openssl-3.0.x/libquery_engine.so.node.gz

**Comment by nurul3101** [comment, marked as answer]
Hey @calidion 👋 
What's your prisma version?
Can you have a look at this [comment](https://github.com/prisma/prisma/issues/11356#issuecomment-1112802322), they were facing a similar issue and they had not updated `@prisma/client`. We are supporting OpenSSL 3.0 from version `3.13.0`.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
